File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/If-Else/leapyear.cpp
#include<iostream>
using namespace std;

int main()
{
    int year;
    cout<<"Enter the year to be checked  "<<endl;
    cin>>year;

    if((year % 400 ==0) || (year % 4 ==0) && (year % 100 !=0))
    cout<<"The entered year "<< year << " is a leap year"<< endl;

    else 
    cout<<"The entered year "<< year << " is a not leap year"<< endl;
}

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/If-Else/LeapYear.java
//Checking whether the year entered by the user is leap year or not 

import java.util.*;


public class LeapYear
{
    public static void main(String[] args) 
    {
        Scanner in= new Scanner(System.in);
        
        System.out.println("Enter the year to be checked ");
        
        int year=in.nextInt();

        if((year % 400 ==0) || (year % 4 ==0) && (year % 100 !=0))
            System.out.println("The entered year " + year + " is a leap year");

        else 
             System.out.println("The entered year " + year + " is a not leap year");
        
    }
}
